controllers/auth.controller.js

```diff
@@ -0,0 +1,53 @@
+const authService = require("../services/auth.service");
+require("dotenv").config();
+
+const registerUser = async (req, res) => {
+
+  if (JSON.stringify(req.body)==="{}") {
+    return res.send({ status: 400, message: "Request body is empty" });
+  }
+  try {
+    
+    const response = await authService.registerUser(req.body);
+
+
+    if (response.status==201) {
+      res.cookie("jsontoken", response.message, { httpOnly: true });
+      response.message = "Registration is successful";
+    
+    } 
+    res.send(response);
+    
+  } catch (err) {
+    if (err.name === "SequelizeUniqueConstraintError") {
+      res.send({ status: 409, message: err.parent.sqlMessage });
+    } else {
+      res.send({ status: 500, message: err });
+    }
+  }
+};
+
+const loginUser = async (req, res) => {
+  try {
+    if (JSON.stringify(req.body)==="{}") {
+      res.send({ status: 400, message: "Request body is empty" });
+    }
+
+    const response = await authService.loginUser(req.body);
+
+   if (response.status==201) {
+      res.cookie("jsontoken", response.message, { httpOnly: true });
```

Comment:
Shouldn't the name of the cookie be "jwttoken" or just "jwt" ?

Reply:
Yes bhai, to find out a bug, I changed this cookie name and forgot to change it. I have just rename it as "jwt".